src/javascript/app/hooks/use-freshChat.js

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
```

Comment:
just a small nit on the file name useFreshchat or something

Reply:
renamed to useFreshChat